src/users/controllers/MeController.js

```diff
@@ -1,17 +1,18 @@
 import { findUserById } from "../repository/findUserByIdRepository.js";
+import { StatusCodes } from "http-status-codes";
 
 export class MeController {
   async handle(req, res) {
     const user = await findUserById(req.userId);
 
     if (!user) {
-      return res.status(404).json({
+      return res.status(StatusCodes.NOT_FOUND).json({
         error: true,
```

Comment:
suas respostas de erro não estão normalizadas. Lembro de ter visto
- `{ error: true, message: x }`
- `{ errorCode: x, message: y }`
- `{ message: x }`

É importante normalizar para que, quem for consumir a api, confie que as informações do erro sempre vão estar no mesmo lugar

Reply:
Opa, anotei aqui como melhoria futura para normalizar! Obrigado pela dica,